src/drivers/gps/gps.cpp

```diff
@@ -416,7 +416,10 @@ void GPS::handleInjectDataTopic()
 {
 	bool updated = false;
 
+	const size_t max_num_injections = 6;
```

Comment:
Magic numbers :-) Can you add a comment why it's 6?

Reply:
Fair point :+1:

Reply:
Problem is I don't remember exactly the argument for 6. I think the injection data that belongs together is split into multiple messages for multiple sattelite networks. So Galileo, Baidu etc.
Let me ask someone who knows for sure.